webui_runner/plugs/wamr_app_aot/shootout/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+ORIG_CWD="$(pwd)/"
+SCRIPT_LOC="$(realpath $(dirname ${BASH_SOURCE:-$0}))"
+LUCET_APP_ROOT="${SCRIPT_LOC}/.."
+SIGHTGLASS_ROOT="${SCRIPT_LOC}/../../.."
+source ${SIGHTGLASS_ROOT}/config.inc
+
+WASM_ENTRY=-DWASM_ENTRY
+WASI_NO_SUPPORT="-DNO_WASI_SUPPORT"
+WASI_CFLAGS="--sysroot=${WASI_SYSROOT}"
+
+WAMRC="${SIGHTGLASS_ROOT}/plugs/wamr_app_aot/build/wamr/wamr-compiler/build/wamrc"
+IWASM="${SIGHTGLASS_ROOT}/plugs/wamr_app_aot/build/wamr/product-mini/platforms/linux/build/iwasm"
+
+#Prepare shootout
+mkdir -p ${SCRIPT_LOC}/benchmark; cd ${SCRIPT_LOC}/benchmark
+cp -r ${SIGHTGLASS_ROOT}/benchmarks/shootout/* .
+ln -s ${WAMRC} wamrc
+
+#Build shootout
+for cfile in ./*.c; do
+    echo ${WASM_CC} ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} -c $cfile -o $(basename -s .c "$cfile").wasm.o
+    ${WASM_CC}  ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} -c $cfile -o $(basename -s .c "$cfile").wasm.o
+
+    echo ${WASM_CC} ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} $(basename -s .c "$cfile").wasm.o -o $(basename -s .c "$cfile").wasm
+    ${WASM_CC} ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} $(basename -s .c "$cfile").wasm.o -o $(basename -s .c "$cfile").wasm
```

Comment:
Why create the .o file separately? Not saying we shouldn't but we might want to explain why.

Reply:
👍 Just using a template based on an earlier version of clang and lucet that required this two step process. I reworked the flags and was able to compile right to wasm that included the correct entry points.